Bottom: 3cbb32e77456133e5cc8da40ab9d72795e588f07
Top:    f64721d74cfba43d9c91793bc52d924837afe4df
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 13:04:31 +0800

Refresh of autofs-5.0.7-fix-wildcard-multi-map-regression.patch

---

diff --git a/modules/lookup_file.c b/modules/lookup_file.c
index 46771b6..9a5bb8b 100644
--- a/modules/lookup_file.c
+++ b/modules/lookup_file.c
@@ -1105,7 +1105,7 @@ do_cache_lookup:
 		if (ap->type == LKP_INDIRECT && *me->key == '*') {
 			ret = cache_update(mc, source, key, me->mapent, me->age);
 			if (!(ret & (CHE_OK | CHE_UPDATED)))
-				me == NULL;
+				me = NULL;
 		}
 		if (me && (me->source == source || *me->key == '/')) {
 			strcpy(mapent_buf, me->mapent);
diff --git a/modules/lookup_ldap.c b/modules/lookup_ldap.c
index d7c2cf6..83e3215 100644
--- a/modules/lookup_ldap.c
+++ b/modules/lookup_ldap.c
@@ -2987,7 +2987,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 		if (ap->type == LKP_INDIRECT && *me->key == '*') {
 			ret = cache_update(mc, source, key, me->mapent, me->age);
 			if (!(ret & (CHE_OK | CHE_UPDATED)))
-				me == NULL;
+				me = NULL;
 		}
 		if (me && (me->source == source || *me->key == '/')) {
 			strcpy(mapent_buf, me->mapent);
diff --git a/modules/lookup_nisplus.c b/modules/lookup_nisplus.c
index 735a1b3..8237a1e 100644
--- a/modules/lookup_nisplus.c
+++ b/modules/lookup_nisplus.c
@@ -579,7 +579,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 		if (ap->type == LKP_INDIRECT && *me->key == '*') {
 			ret = cache_update(mc, source, key, me->mapent, me->age);
 			if (!(ret & (CHE_OK | CHE_UPDATED)))
-				me == NULL;
+				me = NULL;
 		}
 		if (me && (me->source == source || *me->key == '/')) {
 			mapent_len = strlen(me->mapent);
diff --git a/modules/lookup_sss.c b/modules/lookup_sss.c
index cca862a..5c2ed0a 100644
--- a/modules/lookup_sss.c
+++ b/modules/lookup_sss.c
@@ -653,7 +653,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 		if (ap->type == LKP_INDIRECT && *me->key == '*') {
 			ret = cache_update(mc, source, key, me->mapent, me->age);
 			if (!(ret & (CHE_OK | CHE_UPDATED)))
-				me == NULL;
+				me = NULL;
 		}
 		if (me && (me->source == source || *me->key == '/')) {
 			strcpy(mapent_buf, me->mapent);
diff --git a/modules/lookup_yp.c b/modules/lookup_yp.c
index e89660c..3db7584 100644
--- a/modules/lookup_yp.c
+++ b/modules/lookup_yp.c
@@ -680,7 +680,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 		if (ap->type == LKP_INDIRECT && me->key == '*') {
 			ret = cache_update(mc, source, key, me->mapent, me->age);
 			if (!(ret & (CHE_OK | CHE_UPDATED)))
-				me == NULL;
+				me = NULL;
 		}
 		if (me && (me->source == source || *me->key == '/')) {
 			mapent_len = strlen(me->mapent);
